### PR TITLE
app-admin/lnav: Update to 0.11.0 and fix a few bugs

### PR DESCRIPTION
**Pull request by bowlofeggs**
See individual commit messages for details.

### CODE REVIEW

File: app-admin/lnav/lnav-0.11.0.ebuild [hunk 1]
```diff
@@ -12,32 +13,43 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
...
 
 RDEPEND="
 	app-arch/bzip2:0=
 	app-arch/libarchive:=
 	>=dev-db/sqlite-3.9.0
 	dev-libs/libpcre[cxx]
+	pcap? ( net-analyzer/wireshark[tshark] )
 	>=net-misc/curl-7.23.0
-	sys-libs/ncurses:=[unicode(+)?]
+	sys-libs/ncurses
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.11.0.ebuild:26]: This should still be := as we're linked against ncurses, right?

File: app-admin/lnav/lnav-0.11.0.ebuild [hunk 2]
```diff
@@ -12,32 +13,43 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
...
 	sys-libs/readline:0=
 	sys-libs/zlib:0="
-DEPEND="${RDEPEND}"
+# The tests use ssh-keygen and use dsa and rsa keys (which is why ssl is required)
+DEPEND="${RDEPEND}
+	test? ( net-misc/openssh[ssl] )
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.11.0.ebuild:31]: test? (
	dev-cpp/doctest
	net-misc/openssh[ssl]
)

File: app-admin/lnav/lnav-0.11.0.ebuild [hunk 3]
```diff
@@ -12,32 +13,43 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
...
-IUSE="unicode"
+IUSE="pcap test"
+RESTRICT="!test? ( test )"
 
 RDEPEND="
 	app-arch/bzip2:0=
 	app-arch/libarchive:=
 	>=dev-db/sqlite-3.9.0
 	dev-libs/libpcre[cxx]
+	pcap? ( net-analyzer/wireshark[tshark] )
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.11.0.ebuild:24]: Put this at the end as it's conditional? (Usually we do all unconditional sorted, then conditional sorted by USE flag name)

File: app-admin/lnav/lnav-0.11.0.ebuild [hunk 4]
```diff
@@ -12,32 +13,43 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
...
 	>=net-misc/curl-7.23.0
-	sys-libs/ncurses:=[unicode(+)?]
+	sys-libs/ncurses
 	sys-libs/readline:0=
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.11.0.ebuild:27]:
```suggestion
	sys-libs/readline:=
```

File: app-admin/lnav/lnav-0.11.0.ebuild [hunk 5]
```diff
@@ -12,32 +13,43 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
...
 	sys-libs/zlib:0="
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.11.0.ebuild:28]:
```suggestion
	sys-libs/zlib:="
```

File: app-admin/lnav/lnav-0.11.0.ebuild [hunk 6]
```diff
@@ -1,9 +1,10 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit autotools
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.11.0.ebuild:6]: inherit autotools flag-o-matic

File: app-admin/lnav/lnav-0.11.0.ebuild (hunk 5 above)
Comment (bowlofeggs) [reply app-admin/lnav/lnav-0.11.0.ebuild]: https://github.com/gentoo/gentoo/pull/27130/commits/35f567ad4ca52037616cdbbe5da232f9f9d9ec7c

File: app-admin/lnav/lnav-0.11.0.ebuild (hunk 4 above)
Comment (bowlofeggs) [reply app-admin/lnav/lnav-0.11.0.ebuild]: https://github.com/gentoo/gentoo/pull/27130/commits/35f567ad4ca52037616cdbbe5da232f9f9d9ec7c

**Comment by bowlofeggs** [comment]
@thesamesam Thanks for the review. I believe I have addressed your comments. This is ready for re-review.

**Comment by thesamesam** [comment]
Nice job! I've squashed the commits & merged. Thank you!

**Pull request closed without merging**